### PR TITLE
info: Use parameters defined as global parameters

### LINKED ISSUE #851: swupd info does not respect -p / --path option
**Describe the bug**
When you run `swupd info -p <dirname>`, it prints the information about the host, not the target system.

**To Reproduce**
1. Install a version of Clear in a subdirectory (mount a live image, VM image, use `mkosi`, etc.)
2. Make sure it is different from the host system
3. Run `swupd info -p <dirname>`

**Expected behavior**
Swupd should print the information about the contained system.

**Environment (please complete the following information):**
 - Clear Linux OS Version: 28330 (contained system: 28310)

```
$ swupd info -p clear
Installed version: 28330
Version URL:       https://cdn.download.clearlinux.org/update/
Content URL:       https://cdn.download.clearlinux.org/update/
$ swupd info --help
Installed version: 28330
Version URL:       https://cdn.download.clearlinux.org/update/
Content URL:       https://cdn.download.clearlinux.org/update/
$ cat clear/usr/lib/os-release 
NAME="Clear Linux OS"
VERSION=1
ID=clear-linux-os
ID_LIKE=clear-linux-os
VERSION_ID=28310
PRETTY_NAME="Clear Linux OS"
ANSI_COLOR="1;35"
HOME_URL="https://clearlinux.org"
SUPPORT_URL="https://clearlinux.org"
BUG_REPORT_URL="mailto:dev@lists.clearlinux.org"
PRIVACY_POLICY_URL="http://www.intel.com/privacy"
```

### PR DESCRIPTION
**Pull request by otaviobp**
Mostly because we need to use --path to display information about a chroot

Fixes #851

Signed-off-by: Otavio Pontes <otavio.pontes@intel.com>

### CODE REVIEW

File: src/info.c [hunk 1]
```diff
@@ -30,8 +30,44 @@ void print_update_conf_info()
...
+		return false;
+	}
+
+	return true;
+}
+
 enum swupd_code info_main(int UNUSED_PARAM argc, char UNUSED_PARAM **argv)
 {
+	if (!parse_options(argc, argv)) {
+		return SWUPD_INVALID_OPTION;
```
Comment (castulo) [inline src/info.c]: In the case when the user provided an invalid option `print_help()`, that will be consistent with the rest of the commands.

File: src/info.c [hunk 2]
```diff
@@ -30,8 +30,44 @@ void print_update_conf_info()
...
+static bool parse_options(int argc, char **argv)
+{
+	int ind = global_parse_options(argc, argv, &opts);
+
+	if (ind < 0) {
+		return false;
+	}
+
+	if (argc > ind) {
+		fprintf(stderr, "Error: unexpected arguments\n\n");
```
Comment (castulo) [inline src/info.c]: you can use the `error()` from the logger here, we are already changing that, so it is time to start using that.

File: src/info.c (hunk 1 above)
Comment (otaviobp) [reply src/info.c]: I copied that from update and update didn't have that. I'll add it to info and update.

**Comment by otaviobp** [comment]
Fixed the reported problems

**Pull request merged**